sql/updates/latest-dev.sql

```diff
@@ -5,3 +5,37 @@ DROP FUNCTION IF EXISTS distributed_exec;
 DROP PROCEDURE IF EXISTS refresh_continuous_aggregate(regclass,"any","any");
 
 DROP VIEW IF EXISTS timescaledb_information.continuous_aggregates;
+
+-- Rebuild hypertable invalidation log
+ALTER TABLE IF EXISTS _timescaledb_catalog.continuous_aggs_hypertable_invalidation_log
```

Comment:
You should not use DROP COLUMN in update scripts since this leaves a garbage column and screws up the column indexes. You need to completely recreate the table and copy the old data into the new table.

Reply:
Working on that but still not comfortable with the update tests, hence the draft status. Hopefully working now.